src/hooks/useMarketoForm.js

```diff
@@ -2,7 +2,18 @@ import { useEffect } from 'react';
 
 const useMarketoForm = (munchkinId, id, publishableKey) => {
   useEffect(() => {
-    window.MktoForms2.loadForm('//app-abj.marketo.com', munchkinId, id);
+    window.MktoForms2.loadForm(
+      '//app-abj.marketo.com',
+      munchkinId,
+      id,
+      (form) => {
+        // eslint-disable-next-line no-unused-vars
+        form.onSuccess(function (values, followUpUrl) {
+          location.href = 'https://developer.newrelic.com/nerd-days-thank-you';
```

Comment:
Instead of using `location.href` we can use a method from Gatsby called `navigate`. 

at the top of this hook, you can add:
```jsx
import { navigate } from "gatsby"
```

and then instead of writing 
```jsx
location.href='https://developer.newrelic.com/nerd-days-thank-you'
```
we can write: 
```jsx
navigate("/nerd-days-thank-you")
```